### ISSUE TITLE
Cover gh-lint

### ISSUE THREAD

**Issue by bebraw** [survivejs/maintenance-book#44]
There's a stub in #42. It won't need much. A paragraph or two.

**Comment by sapegin** [comment]
I’m actually not sure it’s really useful. Do you know anyone using this?

**Comment by bebraw** [comment]
@epoberezkin uses it internally MailOnline.

**Comment by sapegin** [comment]
Looks like the use case is many many repositories created by random people, so you need to check thinks like description on GitHub.

**Comment by bebraw** [comment]
Correct. It's something for organizations that want to ensure people provide the right meta information for projects for example.

**Comment by sapegin** [comment]
Who are these people? :-)

**Comment by bebraw** [comment]
It's not a mainstream idea yet. We can leave the issue open for now and decide later. It's not primary content for sure.

**Comment by epoberezkin** [comment]
We don't just check meta - more about checking guidelines. E.g., branches are protected, PRs are approved before they are merged, commits are connected to github users, etc.

**Comment by sapegin** [comment]
@epoberezkin But what’s the scale of the project / team where you actually need this?

**Comment by epoberezkin** [comment]
I think if the team is bigger than 20 people it becomes useful... We have 100 different repos updated every week, 200 every month.
